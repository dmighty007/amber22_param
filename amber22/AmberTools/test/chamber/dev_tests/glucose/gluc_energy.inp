* One glucose, get energy and make psf and xplor psf
* M Crowley
*

if @?mingluc .eq. 0 set mingluc no

if @ext .eq. yes ioformat extended
read rtf card name   @top
read param card name @par

read sequence aglc 1
generate gluc first none last none
rename resn bglc select resid 1 end
autogenerate angl dihe

write psf card name gluc.psf
write psf card xplor name gluc.psfx

read coor pdb name glucmin.pdb

if @mingluc .eq. yes goto mingluc


!PRNLev 10

!Debugging
!ANALys TERM NONbond
energy cutnb 999.0  ctofnb 998.0   ctonnb 997.0 switch


!frcdump option
open unit 20 form write name gluc_forces.dat
frcdump unit 20
close unit 20

stop




label mingluc

mini sd nstep 100 nprint 10
write coor pdb name glucmin_new.pdb

stop


