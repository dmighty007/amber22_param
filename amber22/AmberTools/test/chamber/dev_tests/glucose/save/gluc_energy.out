1
                 Chemistry at HARvard Macromolecular Mechanics
                   (CHARMM) - Developmental Version 36a5     August 15, 2010   
       Copyright(c) 1984-2001  President and Fellows of Harvard College
                              All Rights Reserved
      Current operating system: Darwin-10.4.2(x86_64)@mcrowley-127532.nrel.g
                 Created on  3/31/11 at 15:17:21 by user: crowley     

            Maximum number of ATOMS:     60120, and RESidues:       20040
 RDTITL> * THIS IS FOR "center" CHAIN
 RDTITL> * ASSUMES ORIGIN CHAIN IS FIRST IN PSF
 RDTITL> * JFM 12/2003
 RDTITL> *
  
 CHARMM>     
  
 CHARMM>    if @?mingluc .eq. 0 set mingluc no
 Comparing "0" and "0".
 IF test evaluated as true.  Performing command
 Parameter: MINGLUC <- "NO"
  
 CHARMM>     
  
 CHARMM>    read rtf card name ./top_allxx_sugar.inp
 VOPEN> Attempting to open::./TOP_ALLXX_SUGAR.INP::
 MAINIO> Residue topology file being read from unit  90.
 TITLE> * $ID: TOP_ALLXX_SUGAR.INP,V 1.77 2009/02/27 21:09:30 EHATCHER EXP $
 TITLE> * TOPOLOGY FILE FOR CARBOHYDRATES
 TITLE> * IC COORDINATES ARE FOR THERMALIZED STRUCTURES
 TITLE> * GUVENCH, HATCHER, GREENE, AND MACKERELL
 TITLE> *
 **** WARNING from RTFRDR **** The total charge of the residue, GB14,   0.0200000,
 does not equal the expected charge,   0.0000000.
 There were   1 warning(s) from RTFRDR.
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>    read param card name ./par_allxx_sugar.inp
 VOPEN> Attempting to open::./PAR_ALLXX_SUGAR.INP::

          PARAMETER FILE BEING READ FROM UNIT 90
 TITLE> * $ID: PAR_ALLXX_SUGAR.INP,V 1.103 2009/03/09 15:21:50 OGUVENCH EXP $
 TITLE> *>>>>>>>>>>>> ALL-HYDROGEN PARAMETERS USED IN THE <<<<<<<<<<<<<<<<
 TITLE> *>>>>> DEVELOPMENT OF THE CHARMM CARBOHYDRATE  FORCE FIELD<<<<<<<<
 TITLE> *>>>>>>>>>>>>>>>>>>>>>> FEBRUARY 2006 <<<<<<<<<<<<<<<<<<<<<<<<<<<<
 TITLE> *>>>>>>>> DIRECT COMMENTS TO ALEXANDER D. MACKERELL JR. <<<<<<<<<<
 TITLE> *>>>>>>>> 410-706-7442 OR EMAIL:
 TITLE> * INITIAL SUGAR PARAMETER FILE
 TITLE> *
 PARRDR> WARNING: ATOMS IN BOND HCTI HCTI    0.00000   1.51390 DONT EXIST
 PARRDR> WARNING: ATOMS IN BOND OCTI HCTI  450.00000   0.95720 DONT EXIST
 PARRDR> WARNING: ATOMS IN ANGLE HCTI OCTI HCTI   55.00000 104.52000 DONT EXIST
 PARRDR> WARNING: ATOM FOR NBOND HCTI DOESNT EXIST
 PARRDR> WARNING: ATOM FOR NBOND OCTI DOESNT EXIST
*****  WARNING  ***** PARRDR no nonbond parameters for atom type:  99  DUM 
 NO nonbond interactions will be computed for this atom type.
 PARMIO> NONBOND, HBOND lists and IMAGE atoms cleared.
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    read sequence bglc 1
  
 CHARMM>    generate gluc first none last none
 NO PATCHING WILL BE DONE ON THE FIRST RESIDUE
 NO PATCHING WILL BE DONE ON THE LAST  RESIDUE
 GENPSF> Segment   1 has been generated. Its identifier is GLUC.
 PSFSUM> PSF modified: NONBOND lists and IMAGE atoms cleared.
 PSFSUM> Summary of the structure file counters :
         Number of segments      =        1   Number of residues   =        1
         Number of atoms         =       24   Number of groups     =        5
         Number of bonds         =       24   Number of angles     =       42
         Number of dihedrals     =       66   Number of impropers  =        0
         Number of cross-terms   =        0
         Number of HB acceptors  =        6   Number of HB donors  =        5
         Number of NB exclusions =        0   Total charge =   -0.00000
 QXFORM> Expanded format used.
         More than 4 character atom type used.
  
 CHARMM>    autogenerate angl dihe
 AUTOGEN: All angles are removed and regenerated.
 AUTOGEN: All dihedrals are removed and regenerated.
 PSFSUM> PSF modified: NONBOND lists and IMAGE atoms cleared.
 PSFSUM> Summary of the structure file counters :
         Number of segments      =        1   Number of residues   =        1
         Number of atoms         =       24   Number of groups     =        5
         Number of bonds         =       24   Number of angles     =       42
         Number of dihedrals     =       66   Number of impropers  =        0
         Number of cross-terms   =        0
         Number of HB acceptors  =        6   Number of HB donors  =        5
         Number of NB exclusions =        0   Total charge =   -0.00000
  
 CHARMM>     
  
 CHARMM>    write psf card name gluc.psf
 VOPEN> Attempting to open::gluc.psf::
 RDTITL> WRITE PSF CARD XPLOR NAME GLUC.PSFX
 RDTITL> No title read.
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>    write psf card xplor name gluc.psfx
 VOPEN> Attempting to open::gluc.psfx::
 RDTITL>  
 RDTITL> No title read.
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    read coor pdb name glucmin.pdb
 VOPEN> Attempting to open::GLUCMIN.PDB::
  read CHARMM-pdb format
          SPATIAL COORDINATES BEING READ FROM UNIT 90
 TITLE>   THIS IS FOR "center" CHAIN
 TITLE>   ASSUMES ORIGIN CHAIN IS FIRST IN PSF
 TITLE>   JFM 12/2003
 TITLE>    DATE:     6/29/ 7     15:48: 4      CREATED BY USER: RCW
 TITLE>  *
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    if @mingluc .eq. yes goto mingluc
 Parameter: MINGLUC -> "NO"
 Comparing "NO" and "YES".
 IF test evaluated as false.  Skipping command
  
 CHARMM>     
  
 CHARMM>     
  
 CHARMM>    !PRNLev 10
 CHARMM>     
  
 CHARMM>    !Debugging
 CHARMM>    !ANALys TERM NONbond
 CHARMM>    energy cutnb 999.0  ctofnb 998.0   ctonnb 997.0 switch

 NONBOND OPTION FLAGS: 
     ELEC     VDW      ATOMs    CDIElec  SWITch   VATOm    VFSWIt  
     BYGRoup  NOEXtnd  NOEWald 
 CUTNB  =999.000 CTEXNB =999.000 CTONNB =997.000 CTOFNB =998.000
 WMIN   =  1.500 WRNMXD =  0.500 E14FAC =  1.000 EPS    =  1.000
 NBXMOD =      5
 There are        0 atom  pairs and        0 atom  exclusions.
 There are        0 group pairs and        0 group exclusions.
 <MAKINB> with mode   5 found     66 exclusions and     63 interactions(1-4)
 <MAKGRP> found      9 group exclusions.
 Generating nonbond list with Exclusion mode = 5
 == PRIMARY == SPACE FOR      265 ATOM PAIRS AND        0 GROUP PAIRS

 General atom nonbond list generation found:
      210 ATOM PAIRS WERE FOUND FOR ATOM LIST
        0 GROUP PAIRS REQUIRED ATOM SEARCHES

ENER ENR:  Eval#     ENERgy      Delta-E         GRMS
ENER INTERN:          BONDs       ANGLes       UREY-b    DIHEdrals    IMPRopers
ENER EXTERN:        VDWaals         ELEC       HBONds          ASP         USER
 ----------       ---------    ---------    ---------    ---------    ---------
ENER>        0    122.28644      0.00000      7.34823
ENER INTERN>        2.37013      3.32040      0.27722     32.39518      0.00000
ENER EXTERN>        3.61792     80.30558      0.00000      0.00000      0.00000
 ----------       ---------    ---------    ---------    ---------    ---------
  
 CHARMM>     
  
 CHARMM>     
  
 CHARMM>    !frcdump option
 CHARMM>    open unit 20 form write name gluc_forces.dat
 VOPEN> Attempting to open::gluc_forces.dat::
 OPNLGU> Unit 20 opened for WRITE access to gluc_forces.dat
  
 CHARMM>    frcdump unit 20
 Energies and Forces dumped to unit     20
  
 CHARMM>    close unit 20
 VCLOSE: Closing unit   20 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    stop
$$$$$$  New timer profile $$$$$
   List time                       0.00 Other:            0.00
      Nonbond force                   0.00 Other:            0.00
   Energy time                     0.00 Other:            0.00
 Total time                      0.26 Other:            0.26

                    NORMAL TERMINATION BY NORMAL STOP
                    NO WARNINGS WERE ISSUED

                    $$$$$ JOB ACCOUNTING INFORMATION $$$$$
                     ELAPSED TIME:     0.26  SECONDS 
                         CPU TIME:     0.26  SECONDS 
